src/legacy/core_plugins/kibana/public/dashboard/np_ready/dashboard_app_controller.tsx

```diff
@@ -503,14 +549,21 @@ export class DashboardAppController {
     $scope.onFiltersUpdated = filters => {
       // The filters will automatically be set when the queryFilter emits an update event (see below)
       queryFilter.setFilters(filters);
+      updateNavBar();
     };
 
     $scope.onQuerySaved = savedQuery => {
```

Comment:
Do we still need all of this stuff on the scope? Seems like it's only handled in react now.